src/main/java/com/nerodesk/om/mock/MkUser.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2015, nerodesk.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the nerodesk.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.nerodesk.om.mock;
+
+import com.nerodesk.om.Docs;
+import com.nerodesk.om.User;
+import java.io.File;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+/**
+ * Mocked version of user.
+ *
+ * @author Yegor Bugayenko (yegor@teamed.io)
+ * @version $Id$
+ * @since 0.2
+ */
+@ToString
+@EqualsAndHashCode
+public final class MkUser implements User {
+
+    /**
+     * Directory.
+     */
+    private final transient File dir;
+
+    /**
+     * URN.
+     */
+    private final transient String name;
+
+    /**
+     * Ctor.
+     * @param file Directory
+     * @param urn URN
+     */
+    public MkUser(final File file, final String urn) {
+        this.dir = file;
+        this.name = urn;
+    }
+
+    @Override
+    public Docs docs() {
+        return new MkDocs(this.dir, this.name.replace("[^a-z0-9]", "/"));
```

Comment:
Lets replace with `return new MkDocs(this.dir, this.name.replaceAll("[^a-z0-9]", "/"));`